src/System.Drawing.Common/tests/mono/System.Drawing/BitmapTests.cs

```diff
@@ -0,0 +1,1596 @@
+                                Assert.Equal(97, data[n++]);
+                                break;
+                            case PixelFormat.Format16bppArgb1555:
+                                Assert.Equal(130, data[n++]);
+                                Assert.Equal(160, data[n++]);
+                                Assert.Equal(195, data[n++]);
+                                Assert.Equal(176, data[n++]);
+                                break;
+                            case PixelFormat.Format16bppRgb555:
+                                Assert.Equal(130, data[n++]);
+                                Assert.Equal(32, data[n++]);
+                                Assert.Equal(195, data[n++]);
+                                Assert.Equal(48, data[n++]);
+                                break;
+                        }
+                    }
+                    else
+                    {
+                        int n = 0;
+                        Assert.Equal(c.B, data[n++]);
+                        Assert.Equal(c.G, data[n++]);
+                        Assert.Equal(c.R, data[n++]);
+                        if (size % 4 == 0)
+                            Assert.Equal(c.A, data[n++]);
+                        Assert.Equal(d.B, data[n++]);
+                        Assert.Equal(d.G, data[n++]);
+                        Assert.Equal(d.R, data[n++]);
+                        if (size % 4 == 0)
+                            Assert.Equal(d.A, data[n++]);
+                    }
+                }
+                finally
+                {
+                    bmp.UnlockBits(bd);
+                }
+            }
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void Format32bppArgb()
+        {
+            FormatTest(PixelFormat.Format32bppArgb);
+        }
+
+        [ActiveIssue(20884)]
+        public void Format32bppRgb()
+        {
+            FormatTest(PixelFormat.Format32bppRgb);
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void Format24bppRgb()
+        {
+            FormatTest(PixelFormat.Format24bppRgb);
+        }
+
+        /* Get the output directory depending on the runtime and location*/
+        public static string getOutSubDir()
+        {
+            string sSub, sRslt;
+
+            if (Environment.GetEnvironmentVariable("MSNet") == null)
+                sSub = "mono/";
+            else
+                sSub = "MSNet/";
+
+            sRslt = Path.GetFullPath(sSub);
+
+            if (Directory.Exists(sRslt) == false)
+                sRslt = "Test/System.Drawing/" + sSub;
+
+            if (sRslt.Length > 0)
+                if (sRslt[sRslt.Length - 1] != '\\' && sRslt[sRslt.Length - 1] != '/')
+                    sRslt += "/";
+
+            return sRslt;
+        }
+
+        // note: this test fails when saving (for the same reason) on Mono and MS.NET
+        //[ConditionalFact(Helpers.GdiplusIsAvailable)]
```

Comment:
Instead of commenting the `ConditionalFact` out should we instead add `ActiveIssue` or delete the test?